### PR TITLE
LRCI-3575 Add workspace-compile-jdk8 to the stable suite

### PR DESCRIPTION
**Pull request by michaelhashimoto**
https://issues.liferay.com/browse/LRCI-3575

### CODE REVIEW

**Comment by michaelhashimoto** [comment]
ci:test:relevant

**Comment by michaelhashimoto** [comment]
ci:test:liferay-sample-workspace:nocompile

**Comment by michaelhashimoto** [comment]
ci:forward

File: workspaces/liferay-sample-workspace/poshi/build.gradle [hunk 1]
```diff
@@ -6,4 +6,14 @@ dependencies {
...
 	poshiPropertiesFile = file("poshi.properties")
+}
+
+repositories {
+	maven {
+		url "https://repository-cdn.liferay.com/nexus/content/groups/public"
+	}
+
+	maven {
+		url "https://repository.liferay.com/nexus/content/groups/public"
```
Comment (gamerson) [inline workspaces/liferay-sample-workspace/poshi/build.gradle:17]: We should be able to remove this one.

**Comment by michaelhashimoto** [comment]
ci:stop

File: workspaces/liferay-sample-workspace/poshi/build.gradle (hunk 1 above)
Comment (michaelhashimoto) [reply workspaces/liferay-sample-workspace/poshi/build.gradle:17]: I'll try it out.  Thanks for the heads up!

**Comment by michaelhashimoto** [comment]
ci:forward

**Pull request closed without merging**